kvstore/memory_test.go

```diff
@@ -208,8 +229,8 @@ func TestMemory_dump_ok(t *testing.T) {
 
 	expectedTbl := `Key                              Val
 ---                              ---
-psa://tenant-1/deadbeef/beefdead {"some": "json"}
-psa://tenant-2/cafecafe/cafecafe [1, 2, 3]
+psa://tenant-1/deadbeef/beefdead [{"some": "json"}]
```

Comment:
I thought this has been changed to psa-iot?

Reply:
You can safely consider it just a random string with a vague scent of PSA :-)

Reply:
From the point of view of the kvstore consistency with the PSA URL schema is completely irrelevant.